app/src/main/java/com/coolapps/yo/tuto4u/HomeActivity.java

```diff
@@ -1,12 +1,31 @@
 package com.coolapps.yo.tuto4u;
 
 import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class HomeActivity extends BaseActivity {
 
+    RecyclerView mockTutorRecycler;
```

Comment:
Do not name this `mockTutorRecycler`. This recyclerView is real, only the TutorData should be mocked. Could you please rename this to `mTutorRecyclerView`? Thank you.

Reply:
Done